.github/workflows/_meta.yml

```diff
@@ -78,6 +81,23 @@ jobs:
         run: |
           echo "run-kind=$RUN_KIND" | tee -a $GITHUB_OUTPUT
 
+      - name: Get the right SHA
```

Comment:
as we discussed: seems we have to do that step somewhere, looks like have it in _meta one of the possibilty, SGTM